src/main/webapp/js/webrtc_adaptor.js

```diff
@@ -733,6 +733,27 @@ function WebRTCAdaptor(initialValues)
 			thiz.remotePeerConnection[streamId].ontrack = function(event) {
 				thiz.onTrack(event, closedStreamId);
 			}
+			
+			const dataChannelOptions = {
+					ordered: true, // do not guarantee order
```

Comment:
comment "do not guarantee order"  is true?

Reply:
comment is deleted